src/mkdocstrings/loggers.py

```diff
@@ -102,9 +105,12 @@ def get_template_path(context: Context) -> str:
     Returns:
         The relative path to the template.
     """
-    template = context.environment.get_template(context.name)
-    if template.filename:
-        return template.filename.rsplit("/templates/", 1)[1]
+    filename = context.environment.get_template(context.name).filename
+    if filename:
+        try:
+            return str(Path(filename).relative_to(base.TEMPLATES_DIR))
```

Comment:
I'm not sure this would work for custom templates, which would exist outside of `base.TEMPLATES_DIR`?

Reply:
Yes, so an absolute path will be used for those. It's good because otherwise it'd be ambiguous.

Reply:
Oh, perfect then :smile: